docs/references/astro/sign-in-store.mdx

```diff
@@ -5,34 +5,66 @@ description: Clerk's $signInStore nanostore provides a convenient way to access
 
 # `$signInStore`
 
-The `$signInStore` store provides a convenient way to access the [`SignIn`](/docs/references/javascript/sign-in/sign-in){{ target: '_blank' }} object, which allows you to check the current state of a sign-in. This is also useful for creating a custom sign-in flow.
+The `$signInStore` store provides a convenient way to access the [`SignIn`](https://clerk.com/docs/references/javascript/sign-in/sign-in) object, which allows you to check the current state of a sign-in. This is also useful for creating a custom sign-in flow.
 
 ## How to use the `$signInStore` store
 
 ### Check the current state of a sign-in
 
 The following example demonstrates how to use the `$signInStore` store to check the current state of a sign-in.
 
-```tsx {{ filename: 'sign-in-step.tsx' }}
-import { useStore } from '@nanostores/react'
-import { $signInStore } from '@clerk/astro/client'
+<CodeBlockTabs options={['React', 'Vue', 'Svelte']}>
+  ```tsx {{ prettier: false, filename: 'sign-in-step.tsx' }}
```

Comment:
Is there a reason we want to turn off prettier for the React code block?

```suggestion
  ```tsx {{ filename: 'sign-in-step.tsx' }}
```